### ISSUE TITLE
Bug: At some point, yari's auto-correction feature doesn't work.

### ISSUE THREAD

**Issue by hochan222** [mdn/yari#4323]
- **Problem: At some point, yari's auto-correction feature doesn't work.**

An example of one of these paths is shown below. 
  - **Note: this is not an issue specific to `/Web/JavaScript/data_structures#_flaws`.**

![image](https://user-images.githubusercontent.com/22424891/127027530-afdc8c36-7bba-4b1c-8b75-8c3ebd54dae5.png)

```
http://127.0.0.1:3000/ko/docs/Web/JavaScript/data_structures#_flaws
```

**Comment by peterbe** [comment]
Next time you get an error, try to look at the terminal to see what the stacktrace might have been. 
Are you on Windows?

One weird thing I noticed is that you had to press "Fix fixable flaws" *twice* because the first time it would correct `href="/en/docs/...` to `href="/en-US/docs/...` and then the second time it fixed other things after the `/docs/...`.

**Comment by hochan222** [comment]
@peterbe  Thank you for your reply. I went through it again and everything works fine for what happened overnight. (The above error does not occur. However, it shows the same phenomenon as yours.)

> Next time you get an error, try to look at the terminal to see what the stacktrace might have been.

I should have been more detailed, but I'm sorry I wasted your time. Next time I will look the detailed logs and environments. However, I didn't realized that logs were recorded in the yari terminal. I'm embarrassed.

<img src="https://user-images.githubusercontent.com/22424891/127160230-3b7fc464-89fd-4680-b2f1-dc9cd4637e06.png" height="150px" />

> Are you on Windows?

I am Mac! MacBook Pro (Retina, 13-inch, Late 2013).

That issue has been resolved, so I'm closing it. Next time I find a bug, I'll figure it out in detail and fix it.
